docs/install/build_from_source.md

```diff
@@ -214,57 +214,79 @@ It is recommended to set environment variable NCCL_LAUNCH_MODE to PARALLEL when
 For example, you can specify using all cores on Linux as follows:
 
 ```bash
-cmake -j$(nproc)
+cmake -j $(nproc) .
 ```
 
 
 #### Recommended for Systems with NVIDIA GPUs and Intel CPUs
 * Build MXNet with `cmake` and install with MKL DNN, GPU, and OpenCV support:
 
 ```bash
-cmake -j USE_CUDA=1 USE_CUDA_PATH=/usr/local/cuda USE_CUDNN=1 USE_MKLDNN=1
+echo "USE_OPENCV = 1" >> ./config.mk
+echo "USE_CUDA = 1" >> ./config.mk
+echo "USE_CUDA_PATH = /usr/local/cuda" >> ./config.mk
+echo "USE_CUDNN = 1" >> ./config.mk
+echo "USE_MKLDNN = 1" >> ./config.mk
+cmake -j $(nproc) USE_CUDA=1 USE_CUDA_PATH=/usr/local/cuda USE_CUDNN=1 USE_MKLDNN=1 .
```

Comment:
this is not how cmake is invoked, and -j is not a valid cmake option.

Reply:
@larroy, I was testing it on an EC2 and it seems to work because -j was added in cmake 3.12 (https://cmake.org/cmake/help/v3.12/release/3.12.html#cmake-3-12-release-notes). But, it may be better to do something compatible with older versions of cmake. What is the correct way that we should be invoking cmake?

Reply:
check how is done in ci/docker/runtime_functions.sh usually:
```
mkdir -p build && cd build
cmake\
    -DCMAKE_CXX_COMPILER_LAUNCHER=ccache \
    -DCMAKE_C_COMPILER_LAUNCHER=ccache \
    -DUSE_CPP_PACKAGE=ON\
    -DUSE_CUDA=OFF\
    -DUSE_OPENMP=OFF\
    -DUSE_OPENCV=ON\
    -DCMAKE_BUILD_TYPE=Debug\
    -GNinja ..
ninja -v
```

I woud say let's not add options like -j which won't work in the CMake that comes with most distros. Also check the comment below regarding a possible wrapping of CMake in python to simplify builds as is done in dev_menu.py

Hope it helps, thanks.